samples/bot-conversation/python/config.py

```diff
@@ -12,4 +12,4 @@ class DefaultConfig:
 
     PORT = 3978
     APP_ID = os.environ.get("MicrosoftAppId", "<<MICROSOFT-APP-ID>>")
-    APP_PASSWORD = os.environ.get("MicrosoftAppPassword", "<<MICROSOFT-APP-PASSWORD>>")
+    APP_PASSWORD = os.environ.get("MicrosoftAppPassword", "<<MICROSOFT-APP-PASSWORD>>")
```

Comment:
I don't know how botframework python sdk consumes these config but seems BOT_TENANTID and BOT_TYPE is missing here, because js sdk would require these config to make the bot work . This is how js sdk works and I'm not sure it can be used as a reference: https://github.com/OfficeDev/teams-toolkit/blob/0c572e750d8c932b4c82f774a88c3e025d00eb93/templates/ts/default-bot/index.ts#L19